### PR TITLE
Update Firewall_Network_FAQ.pst

### PR DESCRIPTION
**Pull request by jpinkel**
Updated Firewall Network image (Jacob Pinkel - Aviatrix)

### CODE REVIEW

File: HowTos/firewall_network_faq.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ What is Aviatrix Firewall Network (FireNet)?
 Aviatrix Firewall Network (FireNet) is a turn key network solution to deploy firewall instances in the 
 cloud, as shown in the diagram below. 
 
-|firewall_network|
+<img width="1274" alt="image" src="https://user-images.githubusercontent.com/69173400/89824008-44939c80-db07-11ea-9971-7fd7490d6088.png">
```
Comment (sxw97g) [inline HowTos/firewall_network_faq.rst:16]: I'm not sure if the image can be pulled from your repo. Suggest you copy the image as a local png file.

Comment (sxw97g) [inline HowTos/firewall_network_faq.rst:16]: I'm not sure if the image can be pulled from your repo. Please copy the png file to the local directory like the other ones.

Comment (sxw97g) [reply HowTos/firewall_network_faq.rst:16]: I don't think you need to change here. You need to copy the new png file to firewall_network_media/ directory.

Comment (jpinkel) [reply HowTos/firewall_network_faq.rst:16]: I can't seem to upload any files into the fire_wall_network_media/directory without "push access". Would you instead suggest uploading the png into an image hosting site and then copying the url into a "new file" within the repository?

**Pull request closed without merging**